src/metabase/util/memoize.cljs

```diff
@@ -0,0 +1,130 @@
+(ns metabase.util.memoize
+  "Copied from clojure.core.memoize."
```

Comment:
This is a simplified port (copy, really) of the parts of clojure.core.memoize needed for implementing LRU caches.

More could be added or we could create a proper CLJS port, but I wanted to get a fix quickly.